Makefile

```diff
@@ -1060,19 +1060,21 @@ help:
 	@echo "   bump-eve-api   bump eve-api in all subprojects"
 	@echo
 	@echo "Commonly used build targets:"
-	@echo "   build-tools      builds linuxkit utilities and installs under build-tools/bin"
-	@echo "   config           builds a bundle with initial EVE configs"
-	@echo "   pkgs             builds all EVE packages"
-	@echo "   pkg/XXX          builds XXX EVE package"
-	@echo "   rootfs           builds default EVE rootfs image (upload it to the cloud as BaseImage)"
-	@echo "   live             builds a full disk image of EVE which can be function as a virtual device"
-	@echo "   live-XXX         builds a particular kind of EVE live image (raw, qcow2, gcp, vdi, parallels)"
-	@echo "   installer-raw    builds raw disk installer image (to be installed on bootable media)"
-	@echo "   installer-iso    builds an ISO installers image (to be installed on bootable media)"
-	@echo "   installer-net    builds a tarball of artifacts to be used for PXE booting"
-	@echo "   verification-raw builds raw disk verification image (to be installed on bootable media)"
-	@echo "   verification-net builds a tarball of artifacts to be used for PXE verification"
-	@echo "   verification-iso builds an ISO verification image (to be installed on bootable media)"
+	@echo "   build-tools          builds linuxkit utilities and installs under build-tools/bin"
+	@echo "   config               builds a bundle with initial EVE configs"
+	@echo "   pkgs                 builds all EVE packages"
+	@echo "   pkg/XXX              builds XXX EVE package"
+	@echo "   rootfs               builds default EVE rootfs image (upload it to the cloud as BaseImage)"
+	@echo "   live                 builds a full disk image of EVE which can be function as a virtual device"
+	@echo "   LIVE_PKG=<pkg> live  updates existing qcow2 disk image of EVE with RW rootfs (ext4) by only copying specified package. This significantly reduced overall build time of the disk image. Used by developers only!"
```

Comment:
This line is by far bigger than others from the help. Does it make sense to break it into two @echo lines to be more terminal friendly?

```
    @echo "   LIVE_PKG=<pkg> live  updates existing qcow2 disk image of EVE with RW rootfs (ext4) by only copying specified package."
    @echo "                        This significantly reduced overall build time of the disk image. Used by developers only!"
```